components/collapsible/basic/src/index.scss

```diff
@@ -1,8 +1,12 @@
 @import '~@schibstedspain/theme-basic/lib/index';
 
+$bdc-collapsible-basic: $c-gray-lightest !default;
+$bd-collapsible-basic: 1px solid $bdc-collapsible-basic !default;
```

Comment:
Are this variables already defined in `theme-basic`?

Reply:
👍  Now they are. Removing them from here soon.

Reply:
cool!